ambassador/ambassador/ir/ircluster.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, ir: 'IR', aconf: Config,
                  location: str,  # REQUIRED
 
                  service: str,   # REQUIRED
+                 cluster_timeout_s: Optional[str]="3",
```

Comment:
Hmmm, why is this an optional string instead of an optional int?? :smile:

Reply:
Why indeed.. I'll change it.